runtimes/eden/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
-authors = ["Eliott Teissonniere <git.eliott@teissonniere.org>"]
-edition = "2021"
+authors.workspace = true
+edition.workspace = true
 name = "runtime-eden"
-version = "2.4.2"
```

Comment:
Not as much important as our other packages, but it would be good to keep the version as others.